### ISSUE TITLE
[Feature Request] Timer for Auto Disconnecting from Active Channel

### ISSUE THREAD

**Issue by Mijinion** [jagrosh/MusicBot#1314]
### Is your feature request related to a problem? Please describe.

There is an auto disconnect after queue is finished feature, but we have no control over that.

### What is your ideal solution to the problem?

It would be nice if we can set a time limit to auto disconnect from an active channel instead of it disconnecting immediately when queue is finished.

### How would this feature be used?

1. Play a song
2. Nothing is queued
3. Song finishes, bot doesn't disconnect until a set time limit, like 5 minutes.
4. A new song is queued up at like 3 mins, resetting that timer.

### Checklist

- [X] I have checked [the documentation](https://jmusicbot.com/) to make sure this feature doesn't already exist
- [x] I have searched for [similar feature requests](https://github.com/jagrosh/MusicBot/labels/enhancement)
- [X] I have confirmed that my requests is **not** on the [list of things that will not be added](https://jmusicbot.com/things-that-wont-be-added)
- [X] I am running the latest version of the bot: ![Release](https://img.shields.io/github/release/jagrosh/MusicBot.svg)

**Comment by GVKAWESOME** [comment]
I agree with @Mijinion I think it would be nice to specify in the config with a parameter of numbers as minutes.
